api/src/test/java/uk/gov/dhsc/htbhf/claimant/ScheduledServiceIntegrationTest.java

```diff
@@ -270,10 +271,13 @@ private void assertChildTurnsOneOnFirstDayEmailPersonalisationMap(PaymentCycle c
 
     private void assertChildTurnsFourEmailPersonalisationMap(PaymentCycle currentCycle, Map personalisationMap) {
         assertCommonPaymentEmailFields(currentCycle, personalisationMap);
+        boolean isPregnant = currentCycle.getExpectedDeliveryDate() != null && currentCycle.getExpectedDeliveryDate().isAfter(LocalDate.now().minusWeeks(12));
```

Comment:
Why after (now - 12) weeks? I might be being thick here...

Reply:
that's the length of time after the due date that we continue to make payments